integrations/access/opsgenie/app.go

```diff
@@ -75,6 +78,15 @@ func NewOpsgenieApp(ctx context.Context, conf *Config) (*App, error) {
 		PluginName: pluginName,
 		conf:       *conf,
 	}
+	teleClient, err := conf.GetTeleportClient(ctx)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	opsgenieApp.accessMonitoringRules = accessmonitoring.NewAccessMonitoringRuleHandler(accessmonitoring.AccessMonitoringRuleHandlerConfig{
```

Comment:
Similarly, we can avoid stuttering with a few renames here.

```suggestion
	opsgenieApp.accessMonitoringRules = accessmonitoring.NewRuleHandler(accessmonitoring.RuleHandlerConfig{
```